modules/ocf_mirrors/files/robots.txt

```diff
@@ -0,0 +1,39 @@
+# robots.txt for mirrors.ocf.berkeley.edu
```

Comment:
What is the reason for adding these rules? And I wonder if we can autogenerate it, to avoid it going out of sync.

Reply:
https://www.google.com/search?q=site%3Amirrors.ocf.berkeley.edu

Mostly for tidiness reasons I guess. Occasionally I see requests from various search engine crawlers crawling actual files, which is not ideal. Some other mirrors also have robots.txt set to disable access to the projects: 

 - https://mirrors.rit.edu/robots.txt
 - https://mirror.clarkson.edu/robots.txt
 - http://debian.csail.mit.edu/robots.txt

Reply:
We can (and want) to have our index page indexed, but the crawlers should not be allowed to go into the directories.